admin/partials/settings/backup-logs.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * File: backup-logs.php
+ *
+ * Backup logs on settings page.
+ *
+ * @link https://www.boldgrid.com
+ * @since SINCEVERSION
+ *
+ * @package    Boldgrid_Backup
+ * @subpackage Boldgrid_Backup/admin/partials/settings
+ * @copyright  BoldGrid
+ * @version    $Id$
+ * @author     BoldGrid <support@boldgrid.com>
+ */
+
+defined( 'WPINC' ) || die;
+
+$checked = Boldgrid_Backup_Admin_Filelist_Analyzer::is_enabled() ? 'checked' : '';
+
+ob_start();
+?>
+<div class="bg-box">
+	<div class="bg-box-top">
+		<?php esc_html_e( 'Backup Logs', 'boldgrid-backup' ); ?>
+	</div>
+	<div class="bg-box-bottom">
+
+		<table class="form-table">
+			<tr>
+				<th>
+					<strong><?php esc_html_e( 'Filelist Analysis', 'boldgrid-backup' ); ?></strong>
+					<p style="font-weight: normal;">
+						<?php esc_html_e( 'Include a filelist analysis log file with each backup. This log file will show you the largest files and directories that were added to your backup, and can be useful in troubleshooting.', 'boldgrid-backup' ); ?>
+					</p>
+				</th>
+				<td>
+					<input type="checkbox" name="<?php echo esc_attr( Boldgrid_Backup_Admin_Filelist_Analyzer::$settings_key ); ?>" value="1" <?php echo esc_attr( $checked ); ?>>
```

Comment:
In releation to the comment above.... Let's say that I were to go troubleshooting an issue later on down the road, and I needed to be able to find the variable being passed through the $_POST array, in order to find this out, I would have to start here, then locate that key value in the Boldgrid_Backup_Admin_Filelist_Analyzer class file before I could search for that variable in the array. It would make further debugging and troubleshooting easier if this just said 
name="backup_filelist_analysis"

Reply:
I would use "filelist_analysis".

Reply:
@jamesros161 I get it, but I don't want to keep repeating the same string everywhere in the code. If we make a change down the road, we'll have to poke around, find and replace - https://en.wikipedia.org/wiki/Don%27t_repeat_yourself

I'm ok with the variable here instead of the string it represents.

Reply:
It sounds cleaner to not reuse a string for the index key, but it is saved in the settings option.  When you implement a variable for a setting array index key, it looks like it can be safely changed, but it cannot in this case.
I suggest using a static string "filelist_analysis".  It is also easier to locate the applicable code using `grep`, as James mentioned.

Reply:
You guys aren't getting Christmas cards this year

Reply:
fixed in https://github.com/BoldGrid/boldgrid-backup/commit/976a13e64e1ca3ae2cc50a64971615fac67333ac